types.go

```diff
@@ -89,6 +89,8 @@ const (
 	RingBuf
 	InodeStorage
 	TaskStorage
+	// MaxMapType - Bound enum of MapTypes, has to be last in enum.
+	MaxMapType
```

Comment:
How is the `features` package supposed to know the value if I don't export it? Should I rather add a "Getter" like function to the ebpf package to retrieve this value so its not exported as an actual MapType enum value?

Reply:
As discussed offline, one solution is to implement the following:
```go
func (_ MapType) Max() MapType {
  return maxMapType
}
```

It's slightly annoying to use since it requires access to a `MapType` variable, but enables things like `if mt >= mt.Max() { ... }`, which is quite elegant. I much prefer this over having an exported `MaxMapType` const. Any other suggestions @lmb?